### ISSUE TITLE
French

### ISSUE THREAD

**Issue by BY944** [stantanasi/streamflix#250]
### Summary

Hi i can’t watch movie because wiflix change address so you can add some other website french like coflic or frenchstream please thank

### Please confirm the following

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by Xx-nanico-xX** [comment]
Regarder dans les commentaires fr 233 pouvez-vous dépanner vous-même ✌️

**Comment by BY944** [comment]
> Regarder dans les commentaires fr #233 pouvez-vous dépanner vous-même ✌️

oui, mais je suis sur firestick donc je crois pas que c’est possible

**Comment by Xx-nanico-xX** [comment]
il y a toujours une possibilité si tu as un smartphone Android  . je vais t'épargner les explications.
version pour téléphone https://www.mediafire.com/file/95mucair8bwiknr/Streamflix_v1.7.8%252839%2529_src.apk/file
version pour TV  https://www.mediafire.com/file/kk0ds35mnqppdfr/streamflix-v1.7.8-only-tv_src.apk/file ✌️

**Comment by BY944** [comment]
> Regarder dans les commentaires fr [#233](https://github.com/stantanasi/streamflix/issues/233) pouvez-vous dépanner vous-même ✌️

Salut tu peux repartager une apk pour tv de la nouvelle adresse s’il te plait ?

**Comment by Xx-nanico-xX** [comment]
> > Regarder dans les commentaires fr [#233](https://github.com/stantanasi/streamflix/issues/233) pouvez-vous dépanner vous-même ✌️
> 
> Salut tu peux repartager une apk pour tv de la nouvelle adresse s'il te plait ?

Salut je peux refaire un nouveau lien mais ça sert à rien les extracteurs de vidéos sont HS quasiment toutes les vidéos ne seront pas lisibles . 
cordialement
